src/compile/unit.ts

```diff
@@ -281,7 +281,7 @@ export class UnitModel extends ModelWithField {
     if (this.encoding.x || this.encoding.y) {
       return 'cell';
     } else {
-      return undefined;
+      return 'view';
```

Comment:
Any reasons why we're adding `view` here?

Reply:
I'm trying to page this in since it's been a while but I believe it's to force a transparent background to be able to capture events. Without a `'view'` style, there is no background specified on the overall view and so brushes only trigger if someone mouses down on a mark.

Reply:
Yes, https://github.com/vega/vega/pull/3480